PCGen-base/code/src/java/pcgen/base/util/CaseInsensitiveMap.java

```diff
@@ -66,7 +66,7 @@
 	 *            The key to be resolved, if necessary
 	 * @return The key used for storing objects in the HashMap
 	 */
-	private Object resolveObject(Object key)
+	private static Object resolveObject(Object key)
```

Comment:
This is not toString() and should not be here.  This also interacts with PR #47 where I question the value of this change given the side effects there.